cou/utils/openstack_to_track_mapping.csv

```diff
@@ -6,39 +6,44 @@ ceph-mon,focal,xena,pacific
 ceph-mon,focal,yoga,quincy
 ceph-mon,jammy,yoga,quincy
 ceph-mon,jammy,zed,quincy
-ceph-mon,jammy,2023.1,quincy
+ceph-mon,jammy,antelope,quincy

```

Comment:
Somehow keeping `2023.1` will not work:

For example: 
`Channel: 3.9/stable for charm 'rabbitmq-server' on series 'jammy' is not supported by COU. Please take a look at the documentation: https://docs.openstack.org/charm-guide/latest/project/charm-delivery.html to see if you are using the right track.
Cannot generate plan for 'cinder-mysql-router'`

Reply:
My guess is that the `_generate_track_mapping` for the `track_mapping`is not converting the string from the OpenStack release to the OpenStackRelease object. I think if you change and start accessing by the object and not by the release codename string might work.

Reply:
not exactly sure why, but it's okay to use the release name rather than the release date? I am not sure if this string will affect channel switching or not, because after `zed` the channel is named by released date `2023.1/stable` etc

Reply:
okay, so that does used to switch the channel, so I will need to roll it back to `2023.1`